src/Corax/Queries/UnaryMatch.Ordinal.cs

```diff
@@ -137,13 +112,15 @@ private static int BinarySearch(TermQueryItem[] item, ReadOnlySpan<byte> value)
             return -1;
         }
 
-        private static unsafe int FillFuncSequenceAllIn(ref UnaryMatch<TInner, TValueType> match, Span<long> matches)
+        [SkipLocalsInit]
+        private static int FillFuncSequenceAllIn(ref UnaryMatch<TInner, TValueType> match, Span<long> matches)
         {
             var value = ((TermQueryItem[])(object)match._value);
             var requiredSizeOfBitset = value.Length / sizeof(byte) + (value.Length % sizeof(byte) == 0 ? 0 : 1);
-            byte* bitsetBuffer = stackalloc byte[requiredSizeOfBitset];
-            new Span<byte>(bitsetBuffer, requiredSizeOfBitset).Clear();
             
+            //We're skipping init here since we will do it below (only when Inner match has values).
+            byte* bitsetBuffer = stackalloc byte[requiredSizeOfBitset];
```

Comment:
Remove this

Reply:
Why? We are clearing it on every iteration.

Reply:
For context: https://github.com/ravendb/ravendb/commit/be9759537e7ea576df518c5eb6f47221298c8223#commitcomment-124651334

There is a big misconception regarding guarantees about initialization (I explained to @maciejaszyk why the documentation is just plain wrong when dealing with C#). C# code is guaranteed to initialize the stackalloc to 0 UNLESS you specify that you are going to be doing it yourself. Incorrect removal of the `locals .init` marker on methods is treated as a compiler bug. https://github.com/dotnet/runtime/issues/43391#issuecomment-708473741

Reply:
Okay